manifests/dev/vsphere-7.0u1/vanilla/deploy/validatingwebhook.yaml

```diff
@@ -51,7 +51,7 @@ rules:
     resources: ["configmaps"]
     verbs: ["get", "list", "watch"]
 ---
-kind: ClusterRoleBinding
+kind: RoleBinding
 apiVersion: rbac.authorization.k8s.io/v1
 metadata:
   name: vsphere-csi-webhook-role-binding
```

Comment:
`RoleBinding` needs a `namespace` in `metadata`. Can you check once more if this works?

Reply:
Added namespace in metadata